agents/backport_agent.py

```diff
@@ -268,6 +269,8 @@ def get_prompt() -> str:
       {{#cve_id}}(a.k.a. {{.}}){{/cve_id}}.
       {{^build_error}}
       Backport upstream fix {{upstream_fix}}.
+      The upstream URL ({{upstream_fix}}) should be added as a comment
+      above the Patch: tag in the spec file.
```

Comment:
I don't think this is necessary (I believe we can use `<UPSTREAM_FIX>` reference without this).